website/static/css/commentpane.css

```diff
@@ -1,42 +1,46 @@
-#commentPane {
+.commentPane {
     position: fixed;
-    z-index: 98; /* Note: Must be lower than Bootstrap modals, BlockUI */
+    z-index: 999; /* Note: Must be lower than Bootstrap modals, BlockUI */
```

Comment:
Why was this reverted? This was changed to `98` in https://github.com/huangginny/osf.io/commit/e3e35bc58e8f21db3f2d95ca05c5ab651d9b33cd in order to fix an overlap issue.